app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(

```

Comment:
I don't think it's an issue, but wouldn't this method's definition fit in 79 characters?